docs/source/tutorial/06-add-more-info.mdx

```diff
@@ -10,13 +10,11 @@ Go back to **LaunchList.graphql**. Your query is already fetching most of the in
 
 Looking at the schema in Studio Sandbox, you can see that `Launch` has a `mission` property, which allows you to get details of the mission. A mission has both a name and a `missionPatch` property, and the `missionPatch` can optionally take a parameter to specify the desired image size.
 
-Because loading a RecyclerView with large images can impact performance, ask for the name and a `SMALL` mission patch. Update your query to look like the following:
```

Comment:
Nit: I think we refer to it as only 'Sandbox'?  (1 line above, in Studio Sandbox)